lmfit/ui/ipy_fitter.py

```diff
@@ -116,12 +107,11 @@ def close(self):
         self.max_checkbox.close()
 
     def _repr_html_(self):
-        box = Box()
+        box = HBox()
         box.children = [self.value_text, self.vary_checkbox,
                         self.min_text, self.min_checkbox,
                         self.max_text, self.max_checkbox]
         display(box)
```

Comment:
I must admit I didnt read carefully about the python Ipython issue.
I had the same problem as in the issue mentioned above, and fixed them for me. Here I'm using Python 3.4 and IPython 3.
Then I only checked with a Python 2.7 virtual env. if that would work, but still using Ipython 3 so I missed the IPython 2.x issue, as I thought it was without the I...
I'll move the Ipython 2 stuff back in.